db.js

```diff
@@ -114,49 +121,35 @@ exports.init = function (sbot, dir, config) {
     })
   }
 
-  function del(key, cb) {
+  function del(msgId, cb) {
     const guard = guardAgainstDuplicateLogs('del()')
     if (guard) return cb(guard)
 
-    // FIXME: this doesn't work anymore after changing base index
-    baseIndex.keyToSeq(key, (err, seq) => {
+    jitdb.all(key(msgId), 0, false, true, (err, results) => {
       if (err) return cb(err)
-      if (seq == null) return cb(new Error('seq is null!'))
+      if (results.length === 0) return cb(new Error('seq is null!'))
```

Comment:
The error message seems not relevant anymore, how about `'cannot delete ' + msgId + ' because it was not found'`